### PR TITLE
WSSQL-137 Adds support for target cluster switching

### PR DESCRIPTION
**Pull request by rpastrana**
Signed-off-by: rpastrana <rodrigo.pastrana@lexisnexis.com>

### CODE REVIEW

**Comment by rpastrana** [comment]
@wangkx  please review

File: esp/services/ws_sql/ws_sqlService.cpp [hunk 1]
```diff
@@ -811,6 +811,23 @@ bool CwssqlEx::onGetRelatedIndexes(IEspContext &context, IEspGetRelatedIndexesRe
...
+	int clusterscount = clusters.length();
+	if (clusterscount > 0)
+	{
+		hashoptions.appendf("\n#OPTION('AllowedClusters', '%s,", targetcluster);
+		ForEachItemIn(i,clusters)
+		{
+			if (!isValidCluster(clusters.item(i)))
+				throw MakeStringException(-1, "Invalid alternate cluster name: %s", clusters.item(i));
+
+			hashoptions.appendf("%s%s", clusters.item(i), i + 1 == clusterscount ? "" : ",");
```
Comment (wangkx) [inline esp/services/ws_sql/ws_sqlService.cpp]: Minor: if you does not append the "," in line 819, you may use hashoptions.appendf(",%s", clusters.item(i)) here.

**Comment by wangkx** [comment]
@rpastrana just a minor comment.

**Comment by wangkx** [comment]
You are right. We should add version number even in request.

**Comment by rpastrana** [comment]
@wangkx please review

**Comment by wangkx** [comment]
@rpastrana looks good now.

**Pull request merged**